lib/qiita/markdown/filters/sanitize.rb

```diff
@@ -47,6 +47,7 @@ def node
           attributes: {
             "a" => [
               "href",
+              "rel",
```

Comment:
@yuku-t In current implementation, if the A element has `rel` attribute, this `ExternalLink` filter overwrites the value to `nofollow` forcibly.